### ISSUE TITLE
Bad performance in draw_openloop_plot.py

### ISSUE THREAD

**Issue by pgq18** [X-Square-Robot/wall-x#75]
Hello. Thanks for your remarkable work!
I finetuned the wallx-flow model using  [lerobot/libero_goal_image](https://huggingface.co/datasets/lerobot/libero_goal_image) dataset and tested it using draw_openloop_plot.py. The output of the script shown below: 

<img width="1500" height="3500" alt="Image" src="https://github.com/user-attachments/assets/622b5bf3-41a1-4942-bcbb-23a1479da9f5" />

It seems that there are some problems during training or test. The training info shown below:

The flow-loss or train loss has decreased to 0.006 and val_loss has also decreased to 0.014. 
I noticed that #35 has some devices for such problem. I have checked the normalization parameters by printing them in the terminal:

<img width="788" height="42" alt="Image" src="https://github.com/user-attachments/assets/1c0f00c0-544c-44a4-85a7-a78163472479" />

Here is my training config.yml: 

[config_qact.yml](https://github.com/user-attachments/files/23844841/config_qact.yml)

Could you please provide some suggestions how to debug it? Thanks for your work again and looking forward to yout reply.
